index/scorch/merge.go

```diff
@@ -274,19 +273,20 @@ type segmentMerge struct {
 // into the root
 func (s *Scorch) mergeSegmentBases(snapshot *IndexSnapshot,
 	sbs []*zap.SegmentBase, sbsDrops []*roaring.Bitmap, sbsIndexes []int,
-	chunkFactor uint32) (uint64, *IndexSnapshot, uint64, error) {
+	chunkFactor uint32) (*IndexSnapshot, uint64, error) {
 	atomic.AddUint64(&s.stats.TotMemMergeBeg, 1)
 
-	var br bytes.Buffer
-
-	cr := zap.NewCountHashWriter(&br)
-
 	memMergeZapStartTime := time.Now()
 
 	atomic.AddUint64(&s.stats.TotMemMergeZapBeg, 1)
```

Comment:
Should we perhaps edit these stat names as well now?

Reply:
Unfortunately, I don't have a better name that pops to mind at the moment.

And, I guess I could make the argument that the codepaths here are merging in-memory segments (SegmentBase instances), so the "Mem" part of these stat names isn't totally unrelated.